src/pybind/mgr/cephadm/module.py

```diff
@@ -352,6 +352,9 @@ def get_daemon_last_config_deps(self, host, name):
 
     def host_needs_daemon_refresh(self, host):
         # type: (str) -> bool
+        if host in self.mgr.offline_hosts:
+            logger.info(f'Host "{host}" marked as offline. Skipping daemon refresh')
```

Comment:
logger.debug... otherwise still willa ppear in ``ceph -W cephadm`` output every time the serve() loop runs